00-Login/README.md

```diff
@@ -2,23 +2,22 @@
 
 The complete guide to getting started with [react-native-auth0](https://github.com/auth0/react-native-auth0) is our [React Native QuickStart](https://auth0.com/docs/quickstart/native/react-native/00-login).
 
-
-
 ## How to run the sample app
 
-Clone the repository and install the dependencies:
+Clone the repository and install the dependencies with [Yarn](https://yarnpkg.com):
```

Comment:
Added a link just in case someone was not familiar and needed a place to get started. This step works with npm but running does not.